etcdctlv3/command/error.go

```diff
@@ -27,6 +27,7 @@ const (
 	ExitError
 	ExitBadConnection
 	ExitInvalidInput // for txn, watch command
+	ExitBadFeature   // provided a valid flag with a bad value
```

Comment:
I think the cmd library we use will catch some of the bad values (like int flag but provided with hello string). I am not sure how to make this comment more accurate.